modules/tinymce/src/themes/silver/main/ts/ui/context/ContextToolbarAnchor.ts

```diff
@@ -94,7 +94,9 @@ const getAnchorSpec = (editor: Editor, mobile: boolean, data: PositionData, posi
     const newAnchor = {
       ...anchor,
       y: bounds.y,
-      height: bounds.height
+      height: bounds.height,
+      x: bounds.x,
```

Comment:
So this fix will be specific to the context toolbars that is nice. Can we add test for that?

Reply:
I didn't add a test or removed the changelog's entry and the old test, because I wanted an opinion from you and @TheSpyder to understand if the new version of the fix is better then the previous one (checkout to `f5d376226c220a49389e3de00d35a8bfe8f66f84` to check it), I still think that `LayoutInsert` should consider the correct boundaries, but if you tell me this solution is better I'll go with the test and the changelog if you think it was better before I'll switch back to the previous version

Reply:
I was assuming that the `bounds` were by definition something more strict that the `anchor` or something that define the max available area, but now that you mentioned I'm realizing that it was just an assumption.

I'll check it better and see if I need a couple of `Math.min` if I can take the `bounds` as is or some other option

Reply:
@TheSpyder  the `Math.min` was needed so I staid with the overwrite of anchor fields, I also added a comment that I'm not sure is needed/clear, let me know what do you think about it

Reply:
@lorenzo-pomili what this suggests to me is you could use `...bounds` instead of `...anchor`, and then keep the Math.min for `x` and `width`.

Reply:
ok, I see, I changed it